Problems/2798/2798.py

```diff
@@ -0,0 +1,21 @@
+#https://www.acmicpc.net/problem/2798
+T, Target =map(int,input().split())
+
+Num = list(map(int,input().split()))
+
+#i, j, k 3개의 서로 다른 수를 뽑자
+result = 0
+for i in Num:
+    for j in Num:
```

Comment:
```suggestion
for i in (0,len(Num)-2):
    for j in (i+1,len(Num)-1):
```
위에처럼 index를 이용하고, 첫번째 카드(i)를 뽑았을 때 2번째/3번째 카드는 굳이 비교 대상이 되지 않아 For에서 비교할 필요가 없어짐. 따라서, 이렇게 하면 run time 시간이 줄어들 수 있어 보임.

Reply:
아 좋은 생각이네요 이거 코드 썻을 당시에는 for문 돌리는거에 대해 제한 둘 생각을 못했어요
보기 더 이쁜? or 정형화 된? for문을 기계적으로 쓰다보니..